lib/fog/azurerm/models/traffic_manager/traffic_manager_profiles.rb

```diff
@@ -0,0 +1,26 @@
+module Fog
+  module TrafficManager
+    class AzureRM
+      # Traffic Manager Profile Collection for TrafficManager Service
+      class TrafficManagerProfiles < Fog::Collection
+        attribute :resource_group
+        model Fog::TrafficManager::AzureRM::TrafficManagerProfile
+
+        def all
+          requires :resource_group
+          traffic_manager_profiles = []
+          service.list_traffic_manager_profiles(resource_group).each do |profile|
+            traffic_manager_profiles << Fog::TrafficManager::AzureRM::TrafficManagerProfile.parse(profile)
+          end
+          load(traffic_manager_profiles)
+        end
+
+        def get(identity)
```

Comment:
Also add "resource_group" as a parameter in argument list.
Sequence should be: 
resource_group, identity

rename "identity" to "traffic_manager_profile_name"